lib/generate-report.js

```diff
@@ -444,6 +456,33 @@ function generateReport(options) {
             );
         });
     }
+    
+    // From https://gist.github.com/remino/1563878
```

Comment:
For calculating times I'd prefer using a library to keep this code as clean as possible, I don't want to make unit test for time calculations, they always go wrong. We can use `momentjs` for it and solve the time calculation with 3 lines of code and a require at the top of the file.

```js
const moment = require('moment');


function formatDuration(ms) {
    return moment.utc(ms).format("HH:mm:ss.SSS")
}
```

Reply:
Sure, I didn't want to add extra dependencies with my PR. Using a library will save some headaches.